signature/src/suite/ed25519_2020.rs

```diff
@@ -35,6 +35,14 @@ impl From<MnemonicLanguage> for bip39::Language {
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub struct Ed25519Signature(pub Vec<u8>);
 
+impl From<Ed25519Signature> for [u8; 64] {
+    fn from(sig: Ed25519Signature) -> [u8; 64] {
+        let mut bytes = [0; 64];
+        bytes[0..64].copy_from_slice(&sig.0[0..64]);
```

Comment:
You might be able to get away with omitting the initial rage but I haven't tested this:
```suggestion
        bytes.copy_from_slice(&sig.0[0..64]);
```